NapatFeature.java

```diff
@@ -0,0 +1,5 @@
+public class NapatFeature {
+    public void feature() {
+        // this is a new feature
+    }
```

Comment:
this feature function is empty, should improve.

Reply:
This feature function is should not empty.

Reply:
I think so kubb

Reply:
it not complete the feature, can you add the feature in your code.